v5.0.0-beta.4 (#27714)
mui-org/material-ui: CHANGELOG.md

### [Versions](https://material-ui.com/versions/)

> ## 5.0.0-beta4

Reviewer: ```suggestion
## 5.0.0-beta.4
```

<!-- generated comparing v5.0.0-beta.3..next -->

_Aug 13, 2021_

Big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:

- Grid's "auto" behaviour has been fixed by @aaronlademann-wf (#27514)
- @ryancogswell contribution fixed navigation in MenuList (#27526)

### `@material-ui/core@5.0.0-beta.4`

#### Breaking changes

- <!-- 07 --> [Grid] Fix "auto" behavior to match natural width of its content (#27514) @aaronlademann-wf

#### Changes

> - <!-- 30 --> [ButtonBase]: Fix tabIndex type (#27684) @kylegach

Reviewer: ```suggestion
- <!-- 30 --> [ButtonBase] Fix tabIndex type (#27684) @kylegach
```

- <!-- 13 --> [MenuList] Fix text navigation (#27526) @ryancogswell
- <!-- 23 --> [Radio] Fix size prop forwarding with custom icons (#27656) @DouglasPds
- <!-- 10 --> [TextField] Password visibility icons ( "visibility" ⇔ "visibility-off" ) should be reversed (#27507) @tonextone
- <!-- 18 --> [ToggleButton] Fix handling of color prop (#27635) @oliviertassinari

### `@material-ui/codemods@5.0.0-beta.4`

- <!-- 20 --> [codemod] Fix filename case sensitive duplicate @oliviertassinari

### `@material-ui/system@5.0.0-beta.4`

- <!-- 28 --> [StyledEngineProvider] Remove unnecessary emotion cache export (#27680) @garronej
- <!-- 11 --> [system] Fix missing filterProps in compose type (#27618) @R-Bower

### `@material-ui/labs@5.0.0-alpha.42`

- <!-- 27 --> [CalendarPicker] Improve contrast between enabled and disabled days (#27603) @nikitabobers

> - <!-- 32 --> [PickersDay] Render `children` if specified (#27462) @abriginets

Reviewer: @eps1lon What do you think about we remove the comments systematically to get the `` working on this page https://github.com/mui-org/material-ui/releases.
```suggestion
- [PickersDay] Render `children` if specified (#27462) @abriginets
```

If you want a reproduction, open https://github.com/mui-org/material-ui/releases/tag/v5.0.0-beta.3 and check the formatting of #27578.

Reviewer 2: The numbers are only correct for the same prefix, yeah. We should change that. 

> What do you think about we remove the comments systematically to get the `` working on the release page?

How do you envision people answer to these types of questions?

Reviewer: @eps1lon By prefix, do you mean what we used under the brackets: the primary label?

I didn't get what you mean by "type of questions". I can frame it differently. The above comment includes a reproduction with the back ticks not being rendered correctly by GitHub on the release page. Removing the comments solve the issue. Which make me wonder what we would lose if we remove them? The gain of removing them seems to fix the GitHub rendering and to make it easier to sort alphabetically the changes.

- <!-- 05 --> [TreeView] Fix TreeItem label overflow (#27585) @LorenzHenk

### Core

- <!-- 16 --> [core] Batch small changes (#27636) @oliviertassinari
- <!-- 06 --> [core] Change range strategy to bump (#27652) @oliviertassinari
- <!-- 24 --> [core] Fix visual regression example images (#27660) @eps1lon
- <!-- 22 --> [core] Remove mocks of require.context in markdown loader (#27406) @eps1lon
- <!-- 09 --> [core] Reduce use CSS when Checkbox disableRipple is set (#27568) @oliviertassinari

> - <!-- 01 --> [l10n] Add Arabic Sudan (ar-SD) locale (#27588) @YassinHussein

Reviewer: This is not a change on the behind the scene work that supports the product (Core), I think that we should move it under the core package header.

- <!-- 08 --> [test] Add coverage for jss-to-styled prefix from filename (#27522) @eps1lon
- <!-- 15 --> [test] Add current behavior for a11y name vs visible name for PickersDay (#27661) @eps1lon
- <!-- 17 --> [test] Dodge double logging in dev mode (#27653) @oliviertassinari
- <!-- 14 --> [test] Enable skipped test fixed by upstream React fix (#27615) @eps1lon
- <!-- 19 --> [theme] Add missed variants in Components interface (#27453) @nikitabobers

> ### Docs

Reviewer: We have used Docs before Core so far because it seems to impact users more directly, hence something they might want to read first.

### [Versions](https://material-ui.com/versions/)

## 5.0.0-beta.4

<!-- generated comparing v5.0.0-beta.3..next -->

_Aug 13, 2021_

Big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:

> - Grid's "auto" behaviour has been fixed by @aaronlademann-wf (#27514)

Reviewer: ```suggestion
-  🐛 Grid's "auto" behaviour has been fixed by @aaronlademann-wf (#27514)
```

> - @ryancogswell contribution fixed navigation in MenuList (#27526)

Reviewer 2: ```suggestion
- @ryancogswell contribution fixed keyboard navigation in MenuList (#27526)
```
Technically it's just "text-based keyboard navigation" but let's be a bit more vague so that people see the urgency of this update

> - @ryancogswell contribution fixed keyboard navigation in MenuList (#27526)

Reviewer: ```suggestion
- ♿ An important bug with the keyboard navigation in MenuList was fixed (#27526) @ryancogswell
```
